### ISSUE TITLE
rtmp_tcurl fix is broken 

### ISSUE THREAD

**Issue by mpisat** [AirenSoft/OvenMediaEngine#492]
Reference: https://github.com/AirenSoft/OvenMediaEngine/issues/458
I just compiled OvenMediaEngine 0.12.2 and tested RTMP push feature. However, it is broken due to the following additions.

```
        // Compatibility with specific RTMP servers
        av_dict_set(&options, "rtmp_flashver", "FMLE/3.0 (compatible; FMSc/1.0)", 0);
        //av_dict_set(&options, "rtmp_flashver", "rtmp_live", 0);
        av_dict_set(&options, "rtmp_tcurl", _format_context->url, 0);
```

Repetitive rtmp_flashver seems to cause a problem with ffmpeg rtmp relay. When I disabled the second rtmp_flashver and compiled (make release && make install) it works.

[2021-08-29 03:34:12.793] E [RtmpPubWorker:17187] FFmpeg | third_parties.cpp:111  | [URLContext: 0x7f1770003500] Unknown connect error (unsupported authentication method?)
[2021-08-29 03:34:12.793] E [RtmpPubWorker:17187] FFmpeg | third_parties.cpp:111  | [URLContext: 0x7f1770003500] Server error: Connection failed: Application rejected connection.
[2021-08-29 03:34:12.793] E [RtmpPubWorker:17187] RTMPWriter | rtmp_writer.cpp:137  | Error opening file. error(-1313558101:Unknown error occurred), filename(rtmp://relayedhost/app/streamname/streamname.1634678/streamname.1634678)

**Comment by Keukhan** [comment]
@mpisat 

 Thank you for reporting :)

I found that Adobe authentication is required if i put the string  "FMLE /3.0 (compatible; FMSc/1.0)" in flashver when connecting.

Can you provide the RTMP server product information for verification?

**Comment by Keukhan** [comment]
Thanks for the reporting.

Removed invalid rtmp_live parameter and applied it to master branch.

Have a good day. :)

**Comment by mpisat** [comment]
Verified the fix on the 0.12.4 version.
